invalid & unused imports in Node.fs

ts2fable@0.5.1
The generated Node.fs files generate these invalid imports and it blows up at runtime even though they are to be used. Several issues. What can we do?
``` fs
let [<Import("*","setTimeout")>] setTimeout: SetTimeout.IExports = jsNative
let [<Import("*","setImmediate")>] setImmediate: SetImmediate.IExports = jsNative
let [<Import("*","NodeJS")>] nodeJS: NodeJS.IExports = jsNative
let [<Import("*","_debugger")>] _debugger: _debugger.IExports = jsNative
```

@alfonsogarciacaro I tested this just now. It fails with or without making it a function. Starting from my template with `dotnet new http2 -n myhttp2`, then uncomment in `Node.fs`:
``` fs
let [<Import("*","setTimeout")>] setTimeout: SetTimeout.IExports = jsNative
```
or as a function:
``` fs
let [<Import("*","setTimeout")>] setTimeout(): SetTimeout.IExports = jsNative
```
I get the same error. As a function, I get this error:
```
Error: Cannot find module 'setTimeout'
    at Function.Module._resolveFilename (module.js:542:15)
    at Function.Module._load (module.js:472:25)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\camer\tmp\myhttp2\dist\Node.js:8:19)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)
```
Looking at Node.js, it has:
``` js
var _setTimeout = require("setTimeout");

var setTimeout_2 = _interopRequireWildcard(_setTimeout);

const setTimeout = exports.setTimeout = setTimeout_2;
```

--- a/src/Node.fs
+++ b/src/Node.fs
@@ -45,7 +45,6 @@ let [<Import("*","domain")>] domain: Domain.IExports = jsNative
 let [<Import("*","constants")>] constants: Constants.IExports = jsNative
 let [<Import("*","v8")>] v8: V8.IExports = jsNative
 let [<Import("*","timers")>] timers: Timers.IExports = jsNative
-let [<Import("*","_debugger")>] _debugger: _debugger.IExports = jsNative
 let [<Import("*","async_hooks")>] async_hooks: Async_hooks.IExports = jsNative
 let [<Import("*","http2")>] http2: Http2.IExports = jsNative
 
@@ -4872,119 +4871,6 @@ module Timers =
         type [<AllowNullLiteral>] IExports =
             abstract __promisify__: unit -> Promise<unit>
             abstract __promisify__: value: 'T -> Promise<'T>
-
-module _debugger =
-
-    type [<AllowNullLiteral>] IExports =
-        abstract Protocol: ProtocolStatic
-        abstract NO_FRAME: float
-        abstract port: float
-        abstract SourceInfo: body: BreakResponse -> string
-        abstract Client: obj
-
-    type [<AllowNullLiteral>] Packet =
-        abstract raw: string with get, set
-        abstract headers: ResizeArray<string> with get, set
-        abstract body: Message with get, set
-
-    type [<AllowNullLiteral>] Message =
-        abstract seq: float with get, set
-        abstract ``type``: string with get, set
-
-    type [<AllowNullLiteral>] RequestInfo =
-        abstract command: string with get, set
-        abstract arguments: obj option with get, set
-
-    type [<AllowNullLiteral>] Request =
-        inherit Message
-        inherit RequestInfo
-
-    type [<AllowNullLiteral>] Event =
-        inherit Message
-        abstract ``event``: string with get, set
-        abstract body: obj option option with get, set
-
-    type [<AllowNullLiteral>] Response =
-        inherit Message
-        abstract request_seq: float with get, set
-        abstract success: bool with get, set
-        /// Contains error message if success === false. 
-        abstract message: string option with get, set
-        /// Contains message body if success === true. 
-        abstract body: obj option option with get, set
-
-    type [<AllowNullLiteral>] BreakpointMessageBody =
-        abstract ``type``: string with get, set
-        abstract target: float with get, set
-        abstract line: float with get, set
-
-    type [<AllowNullLiteral>] Protocol =
-        abstract res: Packet with get, set
-        abstract state: string with get, set
-        abstract execute: data: string -> unit
-        abstract serialize: rq: Request -> string
-        abstract onResponse: (Packet -> unit) with get, set
-
-    type [<AllowNullLiteral>] ProtocolStatic =
-        [<Emit "new $0($1...)">] abstract Create: unit -> Protocol
-
-    type [<AllowNullLiteral>] ScriptDesc =
-        abstract name: string with get, set
-        abstract id: float with get, set
-        abstract isNative: bool option with get, set
-        abstract handle: float option with get, set
-        abstract ``type``: string with get, set
-        abstract lineOffset: float option with get, set
-        abstract columnOffset: float option with get, set
-        abstract lineCount: float option with get, set
-
-    type [<AllowNullLiteral>] Breakpoint =
-        abstract id: float with get, set
-        abstract scriptId: float with get, set
-        abstract script: ScriptDesc with get, set
-        abstract line: float with get, set
-        abstract condition: string option with get, set
-        abstract scriptReq: string option with get, set
-
-    type [<AllowNullLiteral>] RequestHandler =
-        [<Emit "$0($1...)">] abstract Invoke: err: bool * body: Message * res: Packet -> unit
-        abstract request_seq: float option with get, set
-
-    type [<AllowNullLiteral>] ResponseBodyHandler =
-        [<Emit "$0($1...)">] abstract Invoke: err: bool * ?body: obj option -> unit
-        abstract request_seq: float option with get, set
-
-    type [<AllowNullLiteral>] ExceptionInfo =
-        abstract text: string with get, set
-
-    type [<AllowNullLiteral>] BreakResponse =
-        abstract script: ScriptDesc option with get, set
-        abstract ``exception``: ExceptionInfo option with get, set
-        abstract sourceLine: float with get, set
-        abstract sourceLineText: string with get, set
-        abstract sourceColumn: float with get, set
-
-    type [<AllowNullLiteral>] ClientInstance =
-        inherit NodeJS.EventEmitter
-        abstract protocol: Protocol with get, set
-        abstract scripts: ResizeArray<ScriptDesc> with get, set
-        abstract handles: ResizeArray<ScriptDesc> with get, set
-        abstract breakpoints: ResizeArray<Breakpoint> with get, set
-        abstract currentSourceLine: float with get, set
-        abstract currentSourceColumn: float with get, set
-        abstract currentSourceLineText: string with get, set
-        abstract currentFrame: float with get, set
-        abstract currentScript: string with get, set
-        abstract connect: port: float * host: string -> unit
-        abstract req: req: obj option * cb: RequestHandler -> unit
-        abstract reqFrameEval: code: string * frame: float * cb: RequestHandler -> unit
-        abstract mirrorObject: obj: obj option * depth: float * cb: ResponseBodyHandler -> unit
-        abstract setBreakpoint: rq: BreakpointMessageBody * cb: RequestHandler -> unit
-        abstract clearBreakpoint: rq: Request * cb: RequestHandler -> unit
-        abstract listbreakpoints: cb: RequestHandler -> unit
-        abstract reqSource: from: float * ``to``: float * cb: RequestHandler -> unit
-        abstract reqScripts: cb: obj option -> unit
-        abstract reqContinue: cb: RequestHandler -> unit
 
 module Async_hooks =
 

--- a/src/ts2fable.fs
+++ b/src/ts2fable.fs
@@ -105,8 +105,8 @@ if argv |> List.exists (fun s -> s = "splitter.config.js") then // run from buil
 
     // for test-compile
     writeFile ["node_modules/vscode/vscode.d.ts"] "test-compile/VSCode.fs"
-    writeFile ["node_modules/izitoast/dist/izitoast/izitoast.d.ts"] "test-compile/IziToast.fs"
-    // writeFile ["node_modules/izitoast/types/index.d.ts"] "test-compile/IziToast.fs"
+    // writeFile ["node_modules/izitoast/dist/izitoast/izitoast.d.ts"] "test-compile/IziToast.fs"
+    writeFile ["node_modules/izitoast/types/index.d.ts"] "test-compile/IziToast.fs"
     writeFile ["node_modules/electron/electron.d.ts"] "test-compile/Electron.fs"
     writeFile ["node_modules/@types/react/index.d.ts"] "test-compile/React.fs"
     writeFile ["node_modules/@types/mocha/index.d.ts"] "test-compile/Mocha.fs"

--- a/update.ps1
+++ b/update.ps1
@@ -1,1 +1,3 @@
-.\.paket\paket.exe update
+# .\.paket\paket.exe update
+yarn upgrade
+yarn install

